app/src/main/java/com/vero/tasky/agenda/di/WorkManagerInitializer.kt

```diff
@@ -0,0 +1,29 @@
+package com.vero.tasky.agenda.di
+
+import android.content.Context
+import androidx.startup.Initializer
+import androidx.work.Configuration
+import androidx.work.WorkManager
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object WorkManagerInitializer : Initializer<WorkManager> {
+
+    @Provides
+    @Singleton
+    override fun create(@ApplicationContext context: Context): WorkManager {
+        val configuration = Configuration.Builder().build()
+        WorkManager.initialize(context, configuration)
+        return WorkManager.getInstance(context)
+    }
+
+    override fun dependencies(): List<Class<out Initializer<*>>> {
+        return emptyList()
+    }
+}
```

Comment:
I'm pretty sure you don't need this whole class

You are already setting the configuration on the TaskyApplication.kt